examples/grpc_client/load_testing/image_locust.py

```diff
@@ -0,0 +1,72 @@
+import sys
+import os
+from locust import Locust, TaskSet, task, events
+import datetime
+import tensorflow.contrib.util as tf_contrib_util
+
+sys.path.append(os.path.realpath(os.path.join(os.path.realpath(__file__), '../../')))  # noqa
+from grpc_client_utils import INFERENCE_REQUEST
+from images_2_numpy import load_images_from_list
+from grpc_client import get_stub_and_request
+
+
+class MyTaskSet(TaskSet):
+    data_for_requests = []
+    GRPC_ADDRESS = os.environ.get('GRPC_ADDRESS', "URL")
```

Comment:
did you consider getting all these values from args, not from envs?

Reply:
I thought about it, but locust doesn't provide any custom arguments interface so i would have to write some workarounds.
https://github.com/locustio/locust/issues/65